manim/mobject/three_dimensions.py

```diff
@@ -153,6 +156,81 @@ def set_fill_by_checkerboard(self, *colors, opacity=None):
             face.set_fill(colors[c_index], opacity=opacity)
         return self
 
+    def set_fill_by_value(self, axes: "Mobject", colors: Union[Iterable[Color], Color]):
+        """Sets the color of each face of a parametric surface to a color relative to it's z-value
```

Comment:
**Issue :** Though face is the word employed for parametric Surfaces in manim, I think for the end user it's better to use `point`. What do you think ?

Reply:
I agree that "face" is not the best term, but "point" implies that each point on the screen is being colored something different - which is not the case as it is each `mobject` which makes up the surface that is being colored.  What do you think about changing it to `mobject` instead?